metricbeat/module/system/raid/_meta/test/mdstat1

```diff
@@ -0,0 +1,8 @@
+Personalities : [raid0] [raid1] [linear] [multipath] [raid6] [raid5] [raid4] [raid10]
```

Comment:
I recommend storing this under `system/raid/testdata/[testname]/proc/mdstat`. Using `testdata` seems to be conventional to Go; we have it in a few places. And keeping the `proc/mdstat` ensures that we don't have to mess with temporary files which eliminates code and potential problems. To switch between test cases you just change the `raid.mount_point` between `testdata/scenarioA` and `testdata/scenarioB`.

The "[testname]" could be anything "hostA", "scenarioA", "ubuntu16".

Reply:
I changed it to just `/proc/mdstats` inside testdata as I realised one mdstat file should be enough and we can just add more lines. I kept the `testdata` directory inside _meta as we do for other metricsets too to not conflict with package and and "automatic logic to gather files" we have around metricbeat. Like this only the `_meta` directory has to be ignored from all these logics.